### PR TITLE
add: e2e test

### PR DESCRIPTION
**Pull request by ragnika**
Test against claims and disclosureFrames in test-cases folder
- tests that the lib can verify the sd-jwt it issues

### CODE REVIEW

File: src/e2e.spec.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+const testCases = getTestCases();
+
+it.each(testCases)('should be able to issue then verify %s claims', async (testCase) => {
+  const claims = await loadClaims(testCase);
+  const disclosureFrame = await loadDisclosureFrame(testCase);
+
+  const header = { alg: 'ES256' };
+
+  const sdjwt = await issueSDJWT(header, claims, disclosureFrame, {
```
Comment (linasi) [inline src/e2e.spec.ts:27]: @ragnika since all variable data is configurable, wouldn't it make sense to add something that checks output of the sd-jwt?

**Review by linasi — APPROVED** [review]
👍

**Comment by linasi** [comment]
@ragnika also, please update `test` workflow to run these tests in Github actions.

File: src/e2e.spec.ts (hunk 1 above)
Comment (ragnika) [reply src/e2e.spec.ts:27]: @linasi I feel that would be more of a test for the issueSDJWT.

I think that it would make more sense to add the tests there.

**Comment by ragnika** [comment]
> @ragnika also, please update `test` workflow to run these tests in Github actions.

@linasi this test runs when you run `npm run test`. Should I have separated it out?

**Comment by linasi** [comment]
> > @ragnika also, please update `test` workflow to run these tests in Github actions.
> 
> @linasi this test runs when you run `npm run test`. Should I have separated it out?

Nah, all good then. For some reason I thought they are under a separate npm script.

**Pull request merged**